articles/container.html

```diff
@@ -910,7 +911,7 @@ <h3>Append And Delete Container Elements</h3>
 For the latter manner, we must pass a slice as the only variadic argument and
 which must be followed by three dots <code>...</code>.
 We can learn how to call variadic functions from the article
-<a href="function.html#variadic-call">function types and values</a> later.
+<a href="function.htm#variadic-call">function types and values</a> later.

```

Comment:
looks you PR is based on an old revision. Here misss a "l", which I have fixed in a newer revision.